.github/workflows/marketplace-publish.yml

```
@@ -27,8 +27,8 @@ jobs:
           node-version: 18
 
       - run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

This apparently can be simplified as:
```suggestion
          git config user.email "github-actions[bot]@users.noreply.github.com"
```

From https://github.com/orgs/community/discussions/26560#discussioncomment-11480572 :

> `github-actions[bot]@users.noreply.github.com` is enough.